notebooks/development/Build Breaker identification - PIP.ipynb

```diff
@@ -0,0 +1,1603 @@
+{
```

Comment:
<p>Why do you use softmax here?</p>

 _Reply via <a href='https://app.reviewnb.com/thoth-station/notebooks/pull/35/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>I wanted the distribution to highlight the candidate more visually, since there aren't many matches and plenty of those scores will be zero if I am not mistaken.</p>

Reply:
What is the purpose of doing softmax then on entries which have non-zero score value if they show up in the final listing anyway? Is softmax called just because of visualization?

Reply:
Not sure I understand what you mean. What softmax does, in this case, is that it emphasizes the differences. Consider a match of one word in a sample and two words in the other sample. Using simple normalization you would get 1:2 ratio, using softmax you get roughly 1:3.  That's why I chose this instead of simple normalization.
However, If you're pointing out that it would work even with simple normalization or even without modifications to the raw counts, you're probably right.

Reply:
> However, If you're pointing out that it would work even with simple normalization or even without modifications to the raw counts, you're probably right.

 
That was my point - if a simple counting is sufficient enough, let's do it.